dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetFieldValueValidator.java

```diff
@@ -47,24 +51,28 @@ public boolean isValid(DatasetFieldValue value, ConstraintValidatorContext conte
                 && value.getDatasetField().getParentDatasetFieldCompoundValue().getParentDatasetField().getTemplate() != null) {
             lengthOnly = true;
         }
+        
+        if (lengthOnly) {
+            return true;
```

Comment:
Why don't we do 'return true' instead of lengthOnly = true and then checking if(lengthOnly) ?
Looks like we don't need that variable anymore.

Reply:
Ok. You're right.